lib/simulator-xcode-9.js

```diff
@@ -1,12 +1,58 @@
 import SimulatorXcode8 from './simulator-xcode-8';
-import { shutdown as simctlShutdown } from 'node-simctl';
+import { exec } from 'teen_process';
+import log from './logger';
+import { shutdown as simctlShutdown, bootDevice } from 'node-simctl';
+import { waitForCondition } from 'asyncbox';
 
 
+const SIMULATOR_UI_APP_PATTERN = 'Simulator\\.app';
+const SIMULATOR_SHUTDOWN_TIMEOUT = 15 * 1000;
+
 class SimulatorXcode9 extends SimulatorXcode8 {
   constructor (udid, xcodeVersion) {
     super(udid, xcodeVersion);
   }
 
+  set isHeadless (newValue) {
```

Comment:
I didn't want to change the constructor, otherwise it might require additional changes in the client drivers. If we decide to refactor the stuff then I'd like to change scaleFactor implementation and some other additions, which only work before Simulator is booted